nushell.yaml

```diff
@@ -0,0 +1,75 @@
+package:
+  name: nushell
+  version: 0.99.1
+  epoch: 0
+  description: A new type of shell
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cargo-auditable
+      - libgit2-dev
+      - libssh2-dev
+      - openssf-compiler-options
+      - openssl-dev
+      - rust
+      - sqlite-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 2d7c779b90e6382516db74d2a4b902764ae4e739c4b1b0d615521c7d8082c0d5
+      uri: https://github.com/nushell/nushell/archive/${{package.version}}.tar.gz
+
+  # TODO: figure out why I cant link against mimalloc
+  # error: gnu/bin/ld: cannot find -lmimalloc: No such file or directory
+  - runs: |
+      #!/bin/bash
+      set -x
+
+      export LIBSSH2_SYS_USE_PKG_CONFIG=1  # use system libssh2
+      export TARGET=$(rustc -vV | sed -n 's/host: //p')
+      mkdir -p ~/.cargo
+      cat >> ~/.cargo/config.toml <<-EOF
+
+        [target.$TARGET]
+        git2 = { rustc-link-lib = ["git2"] }
+        rusqlite = { rustc-link-lib = ["sqlite3"] }
+      EOF
+
+  - runs: |
+      cargo auditable build --workspace --locked --release
```

Comment:
we can consider using `cargo/build` pipeline [here](https://github.com/chainguard-dev/melange/blob/main/pkg/build/pipelines/cargo/build.yaml)

Reply:
i couldn't use that. in the `target/release`, there are many files but i only want to install some files (executable).

`cargo/build` support either 1 file or all files. 

That's why I still use `uses` pipeline for now.

Reply:
I think it does support multiple files if i set `output` like `nu nu_plugin_inc foo bar` but then, i will have to list all the plugin names there.

do you prefer it that way?

```yaml
- uses: cargo/build
  with:
    opts: --workspace --release --locked
    output: nu nu_plugin_custom_values nu_plugin_example nu_plugins_formats ....
```

Reply:
I think range is not supported as of now. 

ref: https://github.com/chainguard-dev/melange/blob/c072b034edbb302e60599c6789630a3f3153c867/pkg/build/pipelines/cargo/build.yaml#L42